packages/twenty-front/src/modules/ui/layout/page/PageHeader.tsx

```diff
@@ -54,14 +54,23 @@ const StyledTitleContainer = styled.div`
   max-width: 50%;
 `;
 
-const StyledBackIconButton = styled(IconButton)`
+const StyledButtonDiv = styled.div`
+  align-items: center;
+  display: flex;
+  height: 24px;
+  justify-content: center;
   margin-right: ${({ theme }) => theme.spacing(1)};
+  width: 24px;
+`;
+const StyledBackIconButton = styled(IconButton)`
+  height: 24px;
```

Comment:
this actually looks unnecessary if you pass the right size to the IconButton (medium)